go.mod

```diff
@@ -152,3 +154,13 @@ require (
 	sigs.k8s.io/kustomize/kyaml v0.17.1 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.2 // indirect
 )
+
+replace (
+	k8s.io/api => k8s.io/api v0.31.0
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.31.0
+	k8s.io/apimachinery => k8s.io/apimachinery v0.31.0
+	k8s.io/apiserver => k8s.io/apiserver v0.31.0
+	k8s.io/client-go => k8s.io/client-go v0.31.0
+	k8s.io/code-generator => k8s.io/code-generator v0.31.0
+	sigs.k8s.io/structured-merge-diff/v4 => sigs.k8s.io/structured-merge-diff/v4 v4.4.1
+)
```

Comment:
am not a fan of these. Why do we have to override? We should not.

Reply:
k, gonna try force to use 0.31 deps. Tests were failing as it defaulted to 0.32 :/